packages/devextreme/js/__internal/ui/popup/m_popup.ts

```diff
@@ -619,21 +779,17 @@ class Popup<
     return showCloseButton && showTitle;
   }
 
-  _getLocalizationKey(itemType) {
-    return itemType.toLowerCase() === 'done' ? 'OK' : camelize(itemType, true);
-  }
-
-  _getToolbarButtonStylingMode(shortcut) {
-    // @ts-expect-error
+  _getToolbarButtonStylingMode(shortcut: string): string {
+    // @ts-expect-error ts-error
```

Comment:
would be a wonder if not "ts error" :)

Reply:
Improved here: [f12314916353b33f10d4c7ee4c99a008bcb9d6a4]